lib/modules/manager/nuget/util.spec.ts

```diff
@@ -106,6 +106,156 @@ describe('modules/manager/nuget/util', () => {
         'NuGet.Common',
       ]);
     });
+
+    it('reads nuget config file with default registry disabled', async () => {
+      fs.findUpLocal.mockReturnValue(
+        Promise.resolve<string | null>('NuGet.config'),
+      );
```

Comment:
use mockResolvedValue

Reply:
I'm not sure I understand this comment. We do `fs.readLocalFile.mockResolvedValueOnce(` line 114, is that not correct (sorry, I don't have any typescript knownledge)?

Reply:
@viceice Any help please? 😁

Reply:
@viceice Let me know if the latest changes correspond to what you're expecting.